node/src/tls.rs

```diff
@@ -141,7 +141,7 @@ impl Debug for CertFingerprint {
 
 /// Public key fingerprint.
 #[derive(Copy, Clone, DataSize, Deserialize, Eq, Hash, Ord, PartialEq, PartialOrd, Serialize)]
-pub struct KeyFingerprint(Sha512);
+pub struct KeyFingerprint(pub Sha512);
```

Comment:
Can we get away with not making these `pub` somehow? At the very least I believe they should be `pub(crate)` to be in line with the (poorly enforced) style the rest of the code tries to follow.

Reply:
Yep, and to clarify your answer, did you mean to expose the `PublicKey -> KeyFingerprint` function or the `Sha512` newtype?

Reply:
Ah, I think I understand -- look in the history of the file and potentially re-commit the old functions which lived there.